packages/cactus-test-tooling/src/main/typescript/besu/besu-test-ledger.ts

```diff
@@ -145,7 +145,7 @@ export class BesuTestLedger implements ITestLedger {
    * @see https://github.com/hyperledger/besu/blob/1.5.1/config/src/main/resources/dev.json
    */
   public getGenesisAccountPubKey(): string {
-    return "627306090abaB3A6e1400e9345bC60c78a8BEf57";
+    return "0x627306090abaB3A6e1400e9345bC60c78a8BEf57";
```

Comment:
@travis-payne If not already please include in the commit message the research/explanation that we came up with for why this change was necesary (the whole big number investigation we did a while back, there was some GH issue for it in the web3 repo IIRC - a link to that would be great as well)

Sorry if all of this is already in the commit message I didn't check!